chapters/8-annotations.md

```diff
@@ -178,3 +258,25 @@ Example:
     </rdfs:comment>
 </Annotation>
 ```
+
+**8.5.7** YAML: `comment:`
+
+Example:
+
+```text
+comment: |
+  All of the licenses seen in the file, are matching what was seen during manual inspection.
+
+  There are some terms that can influence the concluded license, and some alternatives may be possible,
+  but the concluded license is one of the options.
+```
+
+**8.5.8** JSON: `"comment":`
+
+In JSON format line breaks need to be escaped as `\n`.
```

Comment:
This is accurate, but I wonder if it should be mentioned as a top-level formatting thing rather than for each field. There are going to be a lot of comment fields across the different sections so it might be easier to say it once somewhere else, rather than repeating each time.